exercises/J-functions/exercise2.js

```diff
@@ -1,5 +1,5 @@
 function triple(number) {
-  // complete function here
+  return (number = number * 3);
```

Comment:
you don't need to overwrite the variable. You can just go : return (variable * 3)